pkg/kubectl/cmd/util/factory.go

```diff
@@ -1224,3 +1224,20 @@ func (f *Factory) NewBuilder(thirdPartyDiscovery bool) *resource.Builder {
 
 	return resource.NewBuilder(mapper, typer, resource.ClientMapperFunc(f.ClientForMapping), f.Decoder(true))
 }
+
+func (f *Factory) ResourcesWithPodSpecs() []*meta.RESTMapping {
+	restMaps := []*meta.RESTMapping{}
+	resourcesWithTemplates := []string{"Pod", "ReplicationController", "Deployment", "DaemonSet", "Job", "ReplicaSet"}
```

Comment:
Is `Pod` intended to be on here?  A pod cannot update its resource requirements once its created.  Is this intended for chaining commands together locally before final creation?